opencog/reasoning/RuleEngine/rule-engine-src/pln/PLNCommons.cc

```diff
@@ -84,6 +85,8 @@ Handle PLNCommons::create_bindLink(
     return as_->addLink(BIND_LINK, var_listLink, implicationLink);
 }
 
+<<<<<<< HEAD
+=======
```

Comment:
This is fixed. But after this merge the ForwardChainer unit test breaks again :(